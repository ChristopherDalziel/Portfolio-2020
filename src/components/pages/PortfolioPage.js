import React from "react";
import styled from "@emotion/styled";
import ProjectList from "../ProjectList";
import ProjectFilter from "../ProjectFilter";
import backgroundImage from "../../assets/Web_Background.png";

const PortfolioContainer = styled.main`
  height: 100vh;
  background-image: url(${backgroundImage});
  background-size: cover;
  overflow: scroll;
`;

const PortfolioContent = styled.div`
  height: 50%;
  width: 80%;
`;

const PortfolioPage = () => {
  return (
    <PortfolioContainer>
      <PortfolioContent>
        <ProjectFilter />
        <ProjectList />
      </PortfolioContent>
    </PortfolioContainer>
  );
};

export default PortfolioPage;
