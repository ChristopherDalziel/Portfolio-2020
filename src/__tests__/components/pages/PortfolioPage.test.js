import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import PortfolioPage from "../../../components/pages/PortfolioPage";

test("Should render the Portfolio Page correctly", () => {
  const wrapper = shallow(<PortfolioPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
