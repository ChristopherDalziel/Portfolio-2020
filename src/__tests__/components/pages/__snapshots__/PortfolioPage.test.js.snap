// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Portfolio Page correctly 1`] = `
<Styled(main)>
  <Styled(div)>
    <Connect(ProjectFilters) />
    <Connect(ProjectList) />
  </Styled(div)>
</Styled(main)>
`;
